tests/trestle/core/commands/remove_test.py

```diff
@@ -233,13 +242,25 @@ def test_run_failure_filenotfounderror(tmp_path, sample_catalog_minimal):
     testargs = [
         'trestle', 'remove', '-f', re.sub('my_test_model/', '', str(catalog_def_file)), '-e', 'catalog.metadata'
     ]
-    with patch.object(sys, 'argv', testargs):
-        exitcode = Trestle().run()
-        assert exitcode == 1
+    monkeypatch.setattr(sys, 'argv', testargs)
+    exitcode = Trestle().run()
+    assert exitcode == 1
 
 
-def test_run_failure_plan_execute(tmp_path, sample_catalog_minimal):
+def test_run_failure_plan_execute(
+    tmp_path: pathlib.Path, sample_catalog_minimal: Catalog, monkeypatch: MonkeyPatch, caplog: pytest.LogCaptureFixture
+):
     """Test failure plan execute() in _run on RemoveCmd."""
+    # Plant this specific logged error for failing execution in mock_execute:
```

Comment:
This caplog test is neat. Does it capture both stdout and stderr?

Reply:
There seems to be *capsys* for both stdout and stderr.

Reply:
Nice!